cdap-ui/templates/common/Table.json

```diff
@@ -4,7 +4,7 @@
     "position": [ "group1" ],
     "group1": {
        "display" : "General",
-       "position" : [ "name", "schema", "schema.row.field" ],
+       "position" : [ "name", "schema", "schema.row.field", "case.sensitive.row.field" ],
```

Comment:
Yeah, this is a TableSink only property. TableSource will ignore it, so not nice to show it on the UI.